app/src/main/java/com/adiliqbl/wallpapers/ui/images/ImageDetailsFragment.kt

```diff
@@ -14,17 +21,38 @@ class ImageDetailsFragment : BaseFragment<ImageDetailsViewModel>() {
 
     companion object {
         private const val KEY_IMAGE = "image"
+        private const val KEY_TRANSITION_NAME = "transitionName"
 
         @JvmStatic
-        fun newInstance(image: Image): ImageDetailsFragment {
+        fun newInstance(image: Image, transitionName: String): ImageDetailsFragment {
```

Comment:
@adiliqbl Making an extension function in the BaseActivity/BaseFragment which would create and apply the bundle to the new activity/fragment instance, would reduce the code